Explorer/Assets/DCL/WebRequests/Analytics/Metrics/BandwidthUp.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using UnityEngine.Networking;
+
+namespace DCL.WebRequests.Analytics
+{
+    public class BandwidthUp : IRequestMetric
+    {
+        private ulong bandwidth { get; set; }
+
+        public ulong GetMetric() => bandwidth;
+
+        public void OnRequestStarted(ITypedWebRequest request)
+        {
+            // Do nothing
+        }
+
+        public void OnRequestEnded(ITypedWebRequest request)
+        {
+            if (request.UnityWebRequest.result == UnityWebRequest.Result.Success)
```

Comment:
It will throw an exception if the requests end with an exception and, therefore, `Dispose` is called before. We need to change the caller side a bit:

```
catch (UnityWebRequestException exception)
                {
                    attemptsLeft--;

                    // Make the request no longer usable as all data needed is written into the exception itself
                    exception.UnityWebRequest.Dispose();

                    if (exception.IsIrrecoverableError(attemptsLeft))
                        throw;

                    // Print verbose
                    ReportHub.Log(reportCategory, $"Exception occured on loading {typeof(TWebRequest).Name} from {commonArguments.URL.ToString()}.\n"
                                                  + $"Attempt Left: {attemptsLeft}");
                }
                finally { analyticsContainer.OnRequestFinished(request); }
```